scripts/remove_operator.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
+
+import subprocess
+import json
+import logging
+import os
+
+
+log = logging.getLogger(__name__)
+NAMESPACE = os.getenv("NAMESPACE", "spark")
+cmd = "kubectl --namespace {} get instances.kudo.dev -o json".format(NAMESPACE)
+
+def delete_resource(api, resource):
+    log.info("Deleting {} {}".format(api, resource))
+    subprocess.call("kubectl delete {} {}".format(api, resource), shell=True)
+
+subprocess.getoutput(cmd)
+instances = json.loads(subprocess.getoutput(cmd))
+
+versions = set()
+for instance in instances["items"]:
+    if instance["metadata"]["labels"]["kudo.dev/operator"] == "spark":
+        name = instance["metadata"]["name"]
+        plan = instance["status"]["activePlan"]["name"]
```

Comment:
When I tried to run this script, it failed here with 
```
╰ ./scripts/remove_operator.py
Traceback (most recent call last):
  File "./scripts/remove_operator.py", line 24, in <module>
    plan = instance["status"]["activePlan"]["name"]
KeyError: 'activePlan'
```
Doesn't look like this key exists 
```
$ k get --namespace spark instances.kudo.dev -o json
{
    "apiVersion": "v1",
    "items": [
        {
            "apiVersion": "kudo.dev/v1alpha1",
            "kind": "Instance",
            "metadata": {
                "annotations": {
                    "kudo.dev/last-applied-instance-state": "{\"operatorVersion\":{\"name\":\"spark-0.0.1\"}}"
                },
                "creationTimestamp": "2019-10-01T21:13:29Z",
                "generation": 3,
                "labels": {
                    "controller-tools.k8s.io": "1.0",
                    "kudo.dev/operator": "spark"
                },
                "name": "spark-fc4zjl",
                "namespace": "spark",
                "resourceVersion": "9938",
                "selfLink": "/apis/kudo.dev/v1alpha1/namespaces/spark/instances/spark-fc4zjl",
                "uid": "4ca12234-cc1d-4cde-b1db-7eb43ef2d60c"
            },
            "spec": {
                "operatorVersion": {
                    "name": "spark-0.0.1"
                }
            },
            "status": {
                "aggregatedStatus": {
                    "status": "COMPLETE"
                },
                "planStatus": {
                    "deploy": {
                        "lastFinishedRun": null,
                        "name": "deploy",
                        "phases": [
                            {
                                "name": "deploy-spark",
                                "status": "COMPLETE",
                                "steps": [
                                    {
                                        "name": "deploy",
                                        "status": "COMPLETE"
                                    }
                                ]
                            }
                        ],
                        "status": "COMPLETE"
                    }
                }
            }
        }
    ],
    "kind": "List",
    "metadata": {
        "resourceVersion": "",
        "selfLink": ""
    }
}
```

Reply:
```suggestion
        plan = instance["status"]["planStatus"]["deploy"]["name"]
```
this should do it

Reply:
thanks for catching this, updated.

Reply:
Actually, it looks like it depends on KUDO CLI version. The unchanged script works for me with the following client set up:
```
$> kubectl kudo version
KUDO Version: version.Info{GitVersion:"0.6.0", GitCommit:"c74d2274", BuildDate:"2019-09-09T15:56:45Z", GoVersion:"go1.12.9", Compiler:"gc", Platform:"darwin/amd64"}
```

Reply:
ah, I was afraid the CLI version would affect this. And it's likely that until we get to a 1.x version KUDO version, it will continue breaking. You can disregard my comment for now but just note that the 0.7x version of this json is closer to the DCOS SDK fields. `activePlan` vs `planStatus.deploy`. Eventually this change will need to be made.